docs/core/basics_de.md

```diff
@@ -0,0 +1,197 @@
+Dazu werden die folgenden Bestandteile benötigt:
+
+1. Ein laufendes, fertig aufgesetztes CES.
+2. Ein neues Verzeichnis, das die Dogu-Bestandteile bündelt, z. B. `newdogu`.
+3. Dockerfile: `newdogu/Dockerfile`
+   - Erstellung des Docker-Images für das neue Dogu.
+4. Dogu-Descriptor: `newdogu/dogu.json`
+   - Enthält Informationen, die bei Installation und auch bei jedem Start des Dogus von Bedeutung sind.
+5. Eine Webanwendung
+     - Die Fachanwendung, entweder als Quellcode oder in einer ausführbaren Form.
+   
+### 1) CES aufsetzen
+
+- `git clone https://github.com/cloudogu/ecosystem`
+- `cd ecosystem`
+- `vagrant up`
+
+Anschließend muss das Setup des CES auf `http://192.168.56.2:8080` durchgeführt werden. Dabei muss sich ein kostenloser Account
+bei der Platform `mycloudogu` erstellt werden und das Dogu `nginx` im Dogu-Schritt ausgewählt werden.
+Eine detailliertere Beschreibung wie man das Setup durchführt ist im [Quick-Start-Guide des CES](https://docs.cloudogu.com/de/quickstart/) zu finden.
+
+### 2) Verzeichnis anlegen
+
+Ein Verzeichnis muss angelegt werden, das die Dogu-Dateien aufbewahren soll.
+
+Beispiel: `mkdir containers/newdogu`.
+
+Innerhalb der eben gestarteten VM von `vagrant` ist das Verzeichnis unter `/vargrant/containers/newdogu` zu finden.
+
+### 3) Dockerfile
+
+Wie bereits erwähnt, stellt ein Dogu eine containerisierte Anwendung dar.
+Daher muss ein Container-Image erzeugt werden, das dann zu einem Container instanziiert wird.
+
+- Allgemeine Richtlinien zum Schreiben von Dockerfiles finden sich im Bereich [Docker Builder](https://docs.docker.com/engine/reference/builder/) und im Bereich [Best Practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) von Docker.
+- Dogu-Base-Images:
+  - Für Java-basierte Anwendungen kann das Cloudogu [Java](https://github.com/cloudogu/java)-Base-Image verwendet werden.
+  - Für andere Anwendungen wird das Cloudogu [base](https://github.com/cloudogu/base)-Image verwendet.
+  - Dogu-Base-Images enthalten auch immer die Anwendung `doguctl`, mit der sich viele übliche Aufgaben noch leichter durchführen lassen:
+    - Zugriff auf Konfigurationswerte
+    - Verschlüsselung eigener Konfigurationswerte
+    - Go-Templating
+    - Health-Checks
+- Wenn das Dogu eine Web-Anwendung ist, ist die Zeile `ENV SERVICE_TAGS webapp` notwendig.
+  Dies führt dazu, dass das neue Dogu im Warp-Menü angezeigt wird.
+- Kopieren Sie Ihre Ressourcen in das dogu, falls erforderlich.
+- Das Label `MAINTAINER` dient zur schnelleren Kommunikation bei Supportfällen.
+
+Beispiel:
+
+`containers/newdogu/Dockerfile`
+
+```dockerfile
+FROM registry.cloudogu.com/official/base:3.17.1-1
+
+LABEL NAME="official/newdogu" \
+   VERSION="1.0.0-1" \
+   MAINTAINER="your.maintainer.or.catchall@company.com"
+
+ENV SERVICE_TAGS=webapp \
+    SERVICE_REWRITE='{"pattern": "newdogu", "rewrite": ""}'
```

Comment:
Was tut das? Ist es für ein frisches Dogu hier nötig?

Reply:
It is used for the dev python http server because you can't configure a context path.
I will write a comment for that.